packages/wallet/src/redux/protocols/advance-channel/readme.md

```diff
@@ -2,11 +2,12 @@
 
```

Comment:
Do you mean "Advancing the stage of a channel" for the title?

Reply:
Fixing upstream.